chall07/egarcia-.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"io/ioutil"
+	"strings"
+	"strconv"
+	"log"
+)
+
+func main() {
+	response, err := http.Get("https://chall07.42madrid.com/")
+	contents, err := ioutil.ReadAll(response.Body)
+	body := strings.Split(string(contents), " ")
+	string := strings.Split(string(body[0]), ",")
+	red, err := strconv.Atoi(string[1][2:])
+	green, err := strconv.Atoi(string[2][2:])
+	blue, err := strconv.Atoi(string[3][2:])
```

Comment:
You should ALWAYS handle errors. Especially when you parse an external input. They are there for a reason.